pkg/orchestration/wiring/postgres/plugin.go

```diff
@@ -72,38 +72,33 @@ func New() *WiringPlugin {
 	}
 }
 
-func getRDSDependency(dependencies []wiringplugin.WiredDependency) (wiringplugin.WiredDependency, error) {
-	rdsDependency := []wiringplugin.WiredDependency{}
-	if len(dependencies) > 0 {
-		for _, d := range dependencies {
-			if d.Type == rds.ResourceType {
-				rdsDependency = append(rdsDependency, d)
-			}
-		}
-		if len(rdsDependency) > 1 {
-			return wiringplugin.WiredDependency{}, errors.Errorf("Postgres resources can only depend on one RDS resource type")
-		}
-		if len(rdsDependency) == 1 {
-			return rdsDependency[0], nil
-		}
-	}
-	return wiringplugin.WiredDependency{}, nil
-}
-
 func references(resource *orch_v1.StateResource, context *wiringplugin.WiringContext) ([]smith_v1.Reference, error) {
```

Comment:
This entangler needs some refactoring. It is silly that we need to do this dependency extracting twice. I'll do it later.